sota/routers/medal_router.py

```diff
@@ -31,9 +27,16 @@ def get_medal_by_country(country_code: str):
                 "gold": {"$sum": "$sports.gold"},
                 "silver": {"$sum": "$sports.silver"},
                 "bronze": {"$sum": "$sports.bronze"},
+                "sub_sports": {
+                    "$push": {
+                        "sub_id": "$sports.type_id",
+                        "gold": "$sports.gold",
+                        "silver": "$sports.silver",
+                        "bronze": "$sports.bronze",
+                    }
+                },
```

Comment:
I don't want to be a spoilsport, but I think you missed name of subsports.

Reply:
yeah totally

Reply:
![Output](https://github.com/SPaM-Skill-Issue/sota-backend/assets/39559733/1d382e65-1f7b-4b9e-82d8-73631ee5dea4)

Affirmative. I checked with the API tester, and it doesn't contain `sub_sport_name`, please fix. 🙏